dotcom-rendering/src/web/components/CalloutBlockComponent.importable.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/react';
+import { neutral } from '@guardian/source-foundations';
+import { ExpandingWrapper } from '@guardian/source-react-components-development-kitchen';
+import type { CalloutBlockElementV2 } from '../../types/content';
+import { CalloutBlock } from './Callout/Callout';
+import { Deadline } from './Callout/Deadline';
+
+const ruleStyles = css`
+	border-image: repeating-linear-gradient(
+			to bottom,
+			${neutral[86]},
+			${neutral[86]} 1px,
+			transparent 1px,
+			transparent 4px
+		)
+		13;
+	border-top: 13px solid ${neutral[86]};
+`;
```

Comment:
I think this needs to be there so that the design still has a rule at the top even if not in the expanding wrapper. Is that correct or do we just not want the rule at all if it's part of a standalone callout article?